### ISSUE TITLE
Double check our MEP file with Riksdagens öppna data

### ISSUE THREAD

**Issue by MansMeg** [welfare-state-analytics/riksdagen-corpus#46]
http://data.riksdagen.se/data/ledamoter/

**Comment by ninpnin** [comment]
The data is available form 1990 onwards as a nice CSV file. Should we rather directly use that?

**Comment by MansMeg** [comment]
Does it have all the info we want? Then yes.

**Comment by ninpnin** [comment]
Here's the file #68

**Comment by ninpnin** [comment]
There are some missing values.

**Comment by MansMeg** [comment]
Yes. I guess we dont need phone number email and similar. But the other stuff seem reasonable.

Im not sure about the uppdragsroll either.
